acceptance-tests/spec/email_with_pdf_spec.rb

```diff
@@ -96,7 +96,7 @@ def assert_pdf_contents
       # select
       expect(result).to include('Is your cat watching you')
       expect(result).to include('now?')
-      expect(result).to include('machine answer 3') # TODO known bug using the machine value
```

Comment:
is this not fixed now?

Reply:
the value that you set in a select box is the actaul value so there is no seperate machine answer it seems. if the tests pass on this then this comment was incorrect

Reply:
made a commit